Dependency peer mismatch

The `devDependencies` need to be updated to avoid this the following issue:

```
$ npm i
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/bin/iojs" "/usr/local/bin/npm" "i"
npm ERR! node v4.4.1
npm ERR! npm  v2.14.20
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package eslint@2.5.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer eslint-plugin-babel@3.1.0 wants eslint@>=1.0.0
npm ERR! peerinvalid Peer babel-eslint@5.0.1 wants eslint@<2.0.0

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/quelhas/Developer/seegno/eslint-config-seegno/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -28,11 +28,11 @@
   },
   "homepage": "https://github.com/seegno/eslint-config-seegno#readme",
   "devDependencies": {
-    "babel-eslint": "^5.0.0",
-    "eslint": "^2.2.0",
+    "babel-eslint": "^6.0.0",
+    "eslint": "^2.5.1",
     "eslint-plugin-babel": "^3.1.0",
     "mocha": "^2.4.5",
-    "should": "^8.2.2"
+    "should": "^8.3.0"
   },
   "engines": {
     "node": ">=4"

